torchci/lib/bot/drciBot.ts

```diff
@@ -23,8 +24,12 @@ async function getDrciComment(
   return {id: 0, body: ""};
 }
 
-export function formDrciComment(): string {
-  let body = "hello there!"
+export function formDrciComment(prNum: number): string {
+  const hudUrl = "hud.pytorch.org/pr/" + prNum;
+  let body = "<h1>Helpful Links</h1>\n";
```

Comment:
Did not know that it supported html. I think the main convention is to just use markdown though so you can maybe try changing it to 
`# Helpful Links` or something